library/Vanilla/Models/FsThemeProvider.php

```diff
@@ -123,7 +123,6 @@ public function getName($themeKey): string {
     public function getThemeAddon($themeKey): AddonInterface {
         $theme = $this->addonManager->lookupTheme($themeKey);
         if (!($theme instanceof AddonInterface)) {
-            $theme = $this->addonManager->lookupTheme(ThemeModel::FALLBACK_THEME_KEY);
```

Comment:
Are we sure this patch works?

Reply:
Chris, I do believe this may cause some issues in its removed (when fetching individual assets). Do any of your test cases cover that with an invalid theme?

Reply:
I've reverted the change and added some tests.